packages/cursorless-vscode-e2e/suite/fixtures/recorded/languages/perl/changeRegex.yml

```diff
@@ -0,0 +1,22 @@
+languageId: perl
+command:
+  version: 4
+  spokenForm: change regex
+  action: {name: clearAndSetSelection}
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: regularExpression}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: $str =~ /bananabeast/
+  selections:
+    - anchor: {line: 0, character: 14}
+      active: {line: 0, character: 14}
+  marks: {}
+finalState:
+  documentContents: $str =~ //
```

Comment:
I'd be tempted to include the slashes as well. WDYT @AndreasArvidsson

Reply:
Definitely. That is what we are doing on javascript for example.

Reply:
I agree, slashes should be included. Had no idea how to fix it so left it for later.